deps/build.jl

```diff
@@ -78,4 +96,16 @@ copy_config("logo-32x32.png", juliakspec)
 copy_config("logo-64x64.png", juliakspec)
 
 eprintln("Installing julia kernelspec $spec_name")
-run(`$jupyter kernelspec install --replace --user $juliakspec`)
+if basename(jupyter) == "jupyter"
+    # Remove the commit that added this when https://github.com/jupyter/notebook/issues/448 is closed
```

Comment:
Just say `# Just use "$jupyter kernelspec" when jupyter/notebook#448 is resolved and widely available.`  I don't think I want to bother with a separate commit for this.